lib/cogctl/actions/relay_groups/add.ex

```diff
@@ -2,27 +2,27 @@ defmodule Cogctl.Actions.RelayGroups.Add do
   use Cogctl.Action, "relay-groups add"
 
   def option_spec() do
-    [{:name, :undefined, :undefined, {:string, :undefined}, 'Relay Group name (required)'},
-     {:relay, :undefined, 'relay', {:string, :undefined}, 'Relay name (required)'}]
+    [{:relay_group, :undefined, :undefined, {:string, :undefined}, 'Relay Group name (required)'},
+     # Technically this will just be the first relay
+     {:relays, :undefined, :undefined, {:string, :undefined}, 'Relay names (required)'}]
   end
 
-  def run(options, _args, _config, %{token: nil}=endpoint) do
-    with_authentication(endpoint, &run(options, nil, nil, &1))
-  end
-  def run(options, _args, _config, endpoint) do
-    case convert_to_params(options, [relay: :required,
-                                     name: :required]) do
+  def run(options, args, _config, endpoint) do
+    # At least one relay is required, so we specify that here
+    case convert_to_params(options, [relay_group: :required,
+                                     relays: :required]) do
       {:ok, params} ->
-        do_add(endpoint, params)
+        params = %{params | relays: [params.relays | args]}
```

Comment:
see https://github.com/operable/cogctl/blob/b3987085466b851f88675e53bd7fbfab0944d092/lib/cogctl/actions/relay_groups/create.ex#L7

It's fragile in the sense that if someone were to add arguments, they'd get slurped up into the value for relays, which would be unexpected.